docs/dev/developer-guides/aa.md

```diff
@@ -1,11 +1,8 @@
 # Account abstraction support
 
 ::: warning
-
-Please note that with system update released in Feb 2023, the ergs concept is only used by the VM and the SDK (version `0.13.0` and above) whilest the API layer operates with gas. 
-
-Find [more information in the changelog](../troubleshooting/changelog.md).
-
+- Please note that with system update released in Feb 2023, the ergs concept is only used by the VM and the SDK (version `0.13.0` and above) whilest the API layer operates with gas. 
```

Comment:
Just checked, it is not used anywhere in the SDK (at least searching "erg" doesn't find anything). So we may say that it is used in VM only